doc/nrf/ug_thread_tools.rst

```diff
@@ -224,20 +206,45 @@ To install and configure the OpenThread Border Router using the Docker container
 
       sudo docker network create --ipv6 --subnet fd11:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
 
-#. Download the latest version of the OpenThread Border Router Docker image by running the following command:
+#. Download the latest version of the OpenThread Bourder Router docker container for nRF Connect SDK by running the following command:
```

Comment:
1. Typo: "Bourder"
2. It's no longer "the latest" - "compatible with this version of |NCS|"?
3. We don't download a container, we download an image based on which a container is created.